mobify/progressive-web-scaffold
# Mini Cart: Empty Styles

Add styles and assets for the mini-cart when it's empty.

## Changes
- Add a missing flexbox utility classes
- Add image assets for empty cart
- Style the mini-cart's appearance when it's empty

## How to test-drive this PR
- `npm run dev`
- Preview Merlin's Potions
- Open the mini-cart
- Observe the mini-cart (which in this PR defaults to a permanent empty state) and confirm it is styled like the mockups (the Sketch file)
